packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -68,6 +68,11 @@ abstract class BuildFrameworkCommand extends BuildSubCommand {
       ..addFlag('cocoapods',
         help: 'Produce a Flutter.podspec instead of an engine Flutter.xcframework (recommended if host app uses CocoaPods).',
       )
+      ..addFlag('plugins',
+        defaultsTo: true,
+        help: 'Whether to produce frameworks for the plugins. '
+              'Consider using "--no-plugins" if plugin frameworks are not required.',
```

Comment:
This second sentence doesn't seem to be adding anything; it's just saying to consider using the flag if you want the behavior of the flag. (It also sounds like people should pass it if they have no plugins, which isn't necessary or useful.)

I think this should instead discuss how this would need to be used. E.g., "This is intended for cases where plugins are already being built separately."